### ISSUE TITLE
[1.12] [5.2.9] Compacting Drawers Broken?

### ISSUE THREAD

**Issue by Kris1432** [jaquadro/StorageDrawers#545]
Forge: 2424
Version: 5.2.9

Cause: Maybe reloaded the chunks by going back into them? This isn't just visual, I can't pull out of blank areas.

Picture: https://puu.sh/wWa5c.png

**Comment by angelnc1989** [comment]
Just experienced the same bug. There is also some strange interaction with Extra Utilities 2 Compressed Blocks and Compacting Drawers. Hard to describe what happens.

**Comment by Ricket** [comment]
Right, weird interaction between exu and compacting drawers. Below is my setup. It was sort of working for a while, cobblestone seemed to be going into the bottom compacting drawer, but then the top drawer was not apparently receiving things. I added a filter for the top transfer node to only take double-compressed cobblestone. Now I just logged in and the whole system is broken, the bottom compacting drawer has 1 cobblestone and the top one has 1 compressed (not double) cobblestone, and the bottom two transfer nodes are inserting cobblestone but the compacting drawer seems to just be voiding it (always only 1 cobblestone, if I remove it then 1 gets placed in, never more than 1).

![image](https://user-images.githubusercontent.com/196978/28748590-97b3f910-7489-11e7-99e2-7252d89614d0.png)

**Comment by ghost** [comment]
I'm using the same version.
The compacting drawers are indeed messing up. When reloading the game or returning to a chunk, sometimes nuggets and/or ingots do not show. Sometimes even trying to take something out of the drawer does not work. Sometimes I can take them out, but the icon still shows on the drawer, and sometimes I need to break the drawer to get the items out; when doing that, sometimes the items don't show up until I move the drawer in my inventory to another slot.
I know, a lot of 'sometimes' ;)

Also (but this might not be related?) I have a lot of glitches/visual artefacts on screen that vanish as soon as I remove storage drawers from my mod list. (Glitches show as flashing triangles all over the screen for a moment).
